jest.config.js

```diff
@@ -0,0 +1,16 @@
+"use strict";
```

Comment:
We already have a jest.config.ts.   There is no need for a jest.config.js as well.   Please delete jest.config.js.

Reply:
file deleted